components/module.py

```diff
@@ -0,0 +1,27 @@
+class Module:
+    # instances
+    # module_number = 0
+
+    # **kwargs for json
+    def __init__(self, module_number, **kwargs):
+        self.module_number = module_number
+        module_number += 1
+        self.scenes = []
+        self.chars_on_screen = []
+        self.current_line = 0
+        self.current_scene = 0
+
+    # changed from replay
+    def play_current_line(self):
+        print(current_line)
+        # do advance to next line w/o incrementing
+
+    def advance_to_next_line(self):
```

Comment:
remove the contents of this function and just put `pass`, we are defining this function this week